src/index.html

```diff
@@ -15,8 +15,44 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin ="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#logo">
+    <img src="images/logo.png" alt="Moyo logo" class="logo"/>
+      </a>
+
+    <nav class="nav">
+      <ul class="nav__list">
+        <li class="nav__item">
+      <a href="#apple" class="nav__link link1 is-active">APPLE</a>
```

Comment:
The text inside the 'a' tag should not be in uppercase due to accessibility concerns. Use CSS to transform text appearance.